shared/validate/validate.go

```diff
@@ -546,3 +548,23 @@ func IsPCIAddress(value string) error {
 
 	return nil
 }
+
+func ValidateCompression(value string) error {
```

Comment:
This should probably be renamed to `IsCompressionAlgorithm` for consistency.
A comment should also be added to make `golint` happy. Something like:
`IsCompressionAlgorithm validates whether a value is a valid compression algorithm and is available on the system.`

Reply:
Yes govet will probably complain about stammering in naming if you have `validate.Validate...()` too.